azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/internal/AzureLiveTestUtils.java

```diff
@@ -28,6 +30,7 @@ public static Properties defaultProperties(Properties properties) {
        properties.put("oauth.credential", "password");
        properties.put("oauth.endpoint", "https://login.microsoftonline.com/oauth2/token");
        properties.put(CREDENTIAL_TYPE, CLIENT_CREDENTIALS_SECRET.toString());
+       properties.put(PROPERTY_ZONES, "northeurope,eastus");
```

Comment:
those are the region names according to http://azure.microsoft.com/en-us/regions